std/bitmanip.d

```diff
@@ -1536,6 +1564,7 @@ public:
         return _ptr[0 .. dim];
     }
 
+    ///
     @system unittest
```

Comment:
the opCast overload is supposed to show casting to `sizet[]`, but it shows casting to `void[]`